sfcasts/ep1/tdd.md

```diff
@@ -87,35 +100,36 @@ failing due to a type error on `getSizeDescription()`:
 
 > The return value must be of type string and none is returned.
 
-Do you remember earlier we ran our large dinosaur
-test *before* writing the method and we didn't see our "this is supposed to be
-a large dino" message? Well, we don't see it here either... That's
-because PHP threw an error... and so the `getSizeDescription()` message explodes
-*before* the test can run the `assertSame()` method. It's no big deal and we can
-still use the stack trace to see exactly where things went wrong.
+Do you remember earlier we ran our large dinosaur test *before* writing the
+method and we didn't see our "this is supposed to be a large dino" message?
+Well, we don't see it here either... That's because PHP threw an error... and
+so the `getSizeDescription()` message explodes *before* the test can run the
```

Comment:
For some reason, this feels better: I'm using PHPUnit as the "actor" here that is "doing" things.

```suggestion
so the `getSizeDescription()` message explodes *before* PHPUnit can run the
```